addon/components/rdf-form.hbs

```diff
@@ -1,19 +1,51 @@
 <div class="au-o-grid au-o-grid--small" ...attributes>
-  {{#each this.sections as |section|}}
-    <FormSection
-      class={{@groupClass}}
-      @form={{@form}}
-      @section={{section}}
-      @formStore={{@formStore}}
-      @graphs={{@graphs}}
-      @sourceNode={{@sourceNode}}
-      @forceShowErrors={{@forceShowErrors}}
-      @cacheConditionals={{@cacheConditionals}}
-      @last={{this.isLast this.sections section}}
-      @show={{@show}}
-      @level={{@level}}
-      @useNewListingLayout={{@useNewListingLayout}}
-      {{!Temporary argument so we don't break the css hacks in the LPDC module}}
-    />
-  {{/each}}
-</div>
+  {{#if this.setupRdfForm.isRunning}}
+    <AuLoader />
+  {{else}}
+    {{#each this.topLevelModels as |model|}}
+      {{#if (this.isSectionModel model)}}
+        <FormSection
+          class={{@groupClass}}
+          @form={{@form}}
+          @section={{model}}
+          @formStore={{@formStore}}
+          @graphs={{@graphs}}
+          @sourceNode={{@sourceNode}}
+          @forceShowErrors={{@forceShowErrors}}
+          @cacheConditionals={{@cacheConditionals}}
+          @last={{this.isLast this.sections model}}
+          @show={{@show}}
+          @level={{@level}}
+          @useNewListingLayout={{@useNewListingLayout}}
+          {{!Temporary argument so we don't break the css hacks in the LPDC module}}
+        />
+      {{/if}}
+      {{#if (this.isFieldModel model)}}
+        <div
+          class="au-o-flow au-u-margin-left au-u-margin-right au-u-margin-top"
+        >
+          {{#let
+            (component-for-display-type model.displayType show=@show)
+            as |FormField|
+          }}
+            <FormField
```

Comment:
I haven't looked at the implementation of the validation helper, but do these fields still validate correctly? If not, we probably need the same lookup logic in the helpers package?

Reply:
Good question! Didn't think of adding a field with validations to the toplevel form as in the formbuilder you can only add validations to a field if they are in a section (if you click a form togheter)

The formField code was taken from the form-section code so I made an assumption that it would do everyting the same as in that component.

Will be checking this when picking up the ticket again 🙏🏻